packages/react/src/Bento/Bento.tsx

```diff
@@ -0,0 +1,181 @@
+import React, {ReactHTML, ReactElement} from 'react'
+import clsx from 'clsx'
+import {Icon, IconProps} from '@primer/octicons-react'
+import type {BaseProps} from '../component-helpers'
+import {Heading, Text, Link, HeadingProps, TextProps, LinkProps} from '../'
+
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/bento/colors-with-modes.css'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/bento/base.css'
+import styles from './Bento.module.css'
+
+export type Size = 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge'
+export type ColumnIndex = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12
+
+type ResponsiveColumnIndex = Partial<Record<Size, ColumnIndex>> | ColumnIndex
+type ResponsiveRowIndex = Partial<Record<Size, number>> | number
+
+type Flow = 'row' | 'column'
+type ResponsiveFlow = Partial<Record<Size, Flow>> | Flow
+
+type Align = 'start' | 'center' | 'end'
+type ResponsiveAlign = Partial<Record<Size, Align>> | Align
+
+type BentoProps = React.HTMLAttributes<HTMLDivElement> & BaseProps<HTMLDivElement>
+
+const Root = ({className, ...rest}: BentoProps) => {
+  return <section className={clsx(styles.Bento, className)} {...rest}></section>
+}
+
+type BentoItemProps = {
+  columnStart?: ResponsiveColumnIndex
+  columnSpan?: ResponsiveColumnIndex
+  rowStart?: ResponsiveRowIndex
+  rowSpan?: ResponsiveRowIndex
+  flow?: ResponsiveFlow
+  colorMode?: 'light' | 'dark'
```

Comment:
We have an [enum](https://github.com/primer/brand/blob/04790926c578c535cd39b410441ca8252180b2d8/packages/react/src/ThemeProvider/ThemeProvider.tsx#L3) for these values, could we use those instead please? 

```suggestion
  colorMode?: ColorModesEnum.LIGHT | ColorModesEnum.DARK
```

Reply:
Color modes via the ENUM have been added in the following SHA: https://github.com/primer/brand/pull/390/commits/015e64c3e342f101c896f8259daf2c49feb49558